src/App.css

```diff
@@ -0,0 +1,69 @@
+.purple-bg{
```

Comment:
Are we using this CSS file? Please remove it and use it as a component prop instead.

Reply:
removed